manager/cve_handler.py

```diff
@@ -121,6 +135,63 @@ def _id_query(rh_account_id, synopsis):
         )
 
 
+class MitigatedSystemsView(SystemCvesView):
+    """Database queries for /cves/{cve_id}/mitigated_systems API endpoint"""
+
+    @staticmethod
+    def _full_query(rh_account_id, synopsis):
+        # pylint: disable=singleton-comparison
+        selectables = [fn.COALESCE(CveAccountData.status_id, 0).alias('cve_status_id'),
+                       SystemPlatform.inventory_id,
+                       SystemPlatform.display_name,
+                       SystemPlatform.last_evaluation,
+                       SystemPlatform.advisor_evaluated.alias('rules_evaluation'),
+                       SystemPlatform.last_upload,
+                       Status.id.alias('status_id'),
+                       Status.name.alias('status_name'),
+                       SystemVulnerabilities.status_text.alias('status_text'),
+                       SystemVulnerabilities.mitigation_reason,
+                       SystemVulnerabilities.when_mitigated,
+                       InsightsRule.name.alias('rule_id')]
+        if CYNDI_ENABLED:
+            selectables.append(InventoryHosts.tags)
+            selectables.append(InventoryHosts.updated)
+
+        return (
+            SystemVulnerabilities
```

Comment:
It won't work. `SystemVulnerabilites` in manager = `system_vulnerabilities_active` table. If the system-cve combination is mitigated in both vmaas and rules, it won't be selected because it moves to `_inactive` patition.

Reply:
@jdobes It will we partition on this condiditon `PARTITION BY LIST ((when_mitigated IS NULL OR rule_id IS NOT NULL));` Rule id for systems where rule mitigation is present is still set so these records are not moved to `_inactive` partition.